index.js

```diff
@@ -235,7 +235,24 @@ module.exports.node = (scriptPath, args, options = {}) => {
 
 	const stdio = normalizeStdio.node(options);
 
-	const {nodePath = process.execPath, nodeOptions = process.execArgv} = options;
+	// Filter out `--inspect` & `--inspect-brk` from default `execArgv`.
+	const defaultExecArgv = process.execArgv.filter(arg => {
+		if (
+			arg === '--inspect' ||
+			arg.startsWith('--inspect=') ||
+			arg === '--inspect-brk' ||
+			arg.startsWith('--inspect-brk=')
+		) {
+			return false;
+		}
+
+		return true;
+	});
+
+	const {
+		nodePath = process.execPath,
+		nodeOptions = defaultExecArgv
+	} = options;
```

Comment:
Any thoughts on doing it no matter what? I don't really see any good reason users would want to override it.

Reply:
Sometimes you want to set inspect on a subprocess when your main process is just a small wrapper. We could also tell people to move towards https://nodejs.org/api/inspector.html instead for this functionality.

I'm okay either way, just remember that changing this would be a breaking change.

Reply:
> Sometimes you want to set inspect on a subprocess when your main process is just a small wrapper. 

Yes, it looks to me that could be the case, and this change would then be a breaking change if any users is doing this. Based on this, I personally think only doing this as a default value (what the PR currently does) might be better?